### PR TITLE
CSI-1816 CSI-1704 update sidecars for 1.4.0

### PR DESCRIPTION
**Pull request by oriyarde**

### CODE REVIEW

File: deploy/crds/csi.ibm.com_v1_ibmblockcsi_cr.yaml [hunk 1]
```diff
@@ -47,24 +51,24 @@ spec:
 
   sidecars:
   - name: csi-node-driver-registrar
-    repository: quay.io/k8scsi/csi-node-driver-registrar
-    tag: "v1.2.0"
```
Comment (ArbelNathan) [inline deploy/crds/csi.ibm.com_v1_ibmblockcsi_cr.yaml:51]: what is the meaning of switching from quay?

File: deploy/crds/csi.ibm.com_v1_ibmblockcsi_cr_ocp.yaml [hunk 2]
```diff
@@ -1,67 +0,0 @@
-apiVersion: csi.ibm.com/v1
```
Comment (ArbelNathan) [inline deploy/crds/csi.ibm.com_v1_ibmblockcsi_cr_ocp.yaml:1]: why removing the ocp cr?

Comment (oriyarde) [reply deploy/crds/csi.ibm.com_v1_ibmblockcsi_cr_ocp.yaml:1]: it is a temporary yaml [used for non-x86 ocp](https://www.ibm.com/support/knowledgecenter/SSRQ8T_1.3.0/csi_ug_install_operator_github.html) when the community sidecars were not multi-arch

File: deploy/crds/csi.ibm.com_v1_ibmblockcsi_cr.yaml (hunk 1 above)
Comment (oriyarde) [reply deploy/crds/csi.ibm.com_v1_ibmblockcsi_cr.yaml:51]: it means that the images will be pulled from a different registry (quay.io is [deprecated](https://github.com/kubernetes-csi/csi-release-tools/issues/86))

**Review by ArbelNathan — APPROVED** [review]
LGTM

**Pull request merged**